MapboxNavigation/SimulatedRoute.swift

```diff
@@ -1,6 +1,17 @@
 import Foundation
 import CoreLocation
 
+fileprivate let maximumSpeed: CLLocationSpeed = 30 // ~108 kmh
+fileprivate let minimumSpeed: CLLocationSpeed = 5 // ~18 kmh
+fileprivate var speed: CLLocationSpeed = 30
```

Comment:
This variable no longer appears to be used. (Some functions shadow it with a computed speed.)